src/universal/conf/storage.conf

```diff
@@ -15,4 +15,14 @@ grothendieck {
       # path = ${grothendieck.datadir}"/leveldb/"
     }
   }
+
+  pruning {
+    # Pruning mode tha app should start with:
```

Comment:
"Pruning mode that the application will use"? Or just fix the spelling of 'that'